CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v1.10.4 (Unpublished)
```

Comment:
Feel free to remove `(Unpublished)` already since we should probably release the fix asap.